### DISCUSSION TITLE
How to add a new question and translate that question

### DISCUSSION THREAD

**Discussion by BooLa96** [globaleaks/globaleaks-whistleblowing-software#3513] [Q&A]
Hi everyone. 

I'm trying to create a form, and I put in Spanish all the questions. But when I change the language to English or Valencian, does not appear, only the answer box is visible. Also, I go into text customization and since it's a dropdown, I can't add the new question and translate them.

![Translate_Quest](https://github.com/globaleaks/GlobaLeaks/assets/138581634/295e8f93-7084-4855-97af-619f1f5a7b45)

**Comment by mauriziobulgarini** [comment]
Hi! Don't use text customization for this purpose. In order to translate a question just open the questionnaire editor, switch language and edit it.
